web-app/src/modules/core/routes/Validators/components/ValidatorRow.tsx

```diff
@@ -29,30 +43,46 @@ const ValidatorRow: FC<ValidatorRowProps> = ({ validator }) => {
           </span>{' '}
           / {validator.grade.proposedBlocks.toLocaleString()}
         </td>
-      ) : (
-        <td className="px-2 md:px-4 lg:px-6 py-4 text-center">
-          {validator.auditQualification?.toLocaleString()}
-        </td>
       )}
+      <td className="px-2 md:px-4 lg:px-6 py-4 text-center">
+        {validator.auditQualification?.toLocaleString() ? (
+          <XMarkIcon className="w-5 h-5 text-red-500 inline" style={{ marginTop: '-3px' }} />
+        ) : (
+          <CheckIcon className="w-5 h-5 text-green-500 inline" style={{ marginTop: '-3px' }} />
+        )}
+        {validator.auditQualification?.toLocaleString()}
+      </td>
       <td className="px-2 md:px-4 lg:px-6 py-4 text-center">
         <Vouches vouches={validator.vouches} />
       </td>
       <td className="px-2 md:px-4 lg:px-6 py-4 text-right">
-        {`${validator.currentBid && formatPercentage(validator.currentBid.currentBid)} (${
-          validator.currentBid && validator.currentBid.expirationEpoch.toLocaleString()
-        })`}
+        {validator.currentBid
+          ? `${formatPercentage(validator.currentBid.currentBid)} (${
+              validator.currentBid.expirationEpoch > 10000
+                ? '> 10,000'
+                : validator.currentBid.expirationEpoch.toLocaleString()
+            })`
+          : ''}
+      </td>
+      {validator.inSet && (
+        <td className="px-2 md:px-4 lg:px-6 py-4 text-center">
+          {statusIcons[validator.vfnStatus]}
+        </td>
+      )}
+      <td className="px-2 md:px-4 lg:px-6 py-4 text-right">
+        <Money>{Number(validator.unlocked)}</Money>
       </td>
       <td className="px-2 md:px-4 lg:px-6 py-4 text-right">
         <Money>{Number(validator.balance)}</Money>
       </td>
-      {validator.inSet && (
+      {/*validator.inSet && (
```

Comment:
Is this code intentionally commented out?

Reply:
We’re still determining the most relevant data to display, and that was the least important. I will remove from the code.